home.css

```diff
@@ -0,0 +1,613 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'lato', sans-serif;
+  min-height: 100vh;
+  background-color: #e5e5e527;
+}
+
+.social-media,
+.navbar {
+  display: none;
+}
+
+.box {
+  background: url('./images/desktop.png') no-repeat;
+  background-size: cover;
+  margin-bottom: 0;
+  padding-bottom: 50px;
+}
+
+.hamburger {
+  border: none;
+  background: transparent;
+  position: relative;
+  margin-top: 8%;
+  padding-left: 5%;
+}
+
+.hamburger img {
+  width: 60%;
+  height: 70%;
+  margin-bottom: 80%;
+}
+
+.hello {
+  position: relative;
+  color: #f35826;
+  font-size: 200%;
+  margin-bottom: 0%;
+  padding-left: 5%;
+}
+
+.heading {
+  position: relative;
+  color: #f35826;
+  font-size: 400%;
+  margin-top: 0%;
+  font-weight: bold;
+  padding-left: 5%;
+}
```

Comment:
[OPTIONAL]  It would be nicer if you could use the same font from the original design for the main title of the page (Cocogoose font).